src/Grid/Col/index.js

```diff
@@ -15,18 +35,24 @@ export default function Col({ children, xs, sm, md, lg, offset, style }) {
 Col.propTypes = {
   children: PropTypes.node.isRequired,
   style: PropTypes.shape({}),
-  offset: PropTypes.number,
-  xs: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, "hide"]),
-  sm: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, "hide"]),
+  offsetXs: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]),
+  offsetSm: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]),
+  offsetMd: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]),
+  offsetLg: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11]),
```

Comment:
I added offset for all the devices as well. So that we can define different offsets for different devices when needed